### PR TITLE
Posted order is now arrival order

### PR DESCRIPTION
**Pull request by annando**
The word "posted order" is misleading, since we order the post after the arrival date. So the menu option is now renamed.

### CODE REVIEW

**Comment by AlfredSK** [comment]
Hmm. I know, technically, this is the correct term. But for the user this is totally confusing. There is a "arrival order" and a "commented order" now. Nobody will understand the difference.

**Comment by MrPetovan** [comment]
Agreed with @AlfredSK, "arrival" doesn't mean anything for users. I suggest "Natural Order" or "Timeline Order" vs "Activity Order" because likes are counted as comments as well in the "Commented" order.

**Comment by annando** [comment]
I don't think that "natural" order is the correct description, since this would mean that the commented order is the unnatural order.

In German these tabs are named "Neueste Kommentare" (newest comments)
and "Neueste Beiträge" (newest posts). Possibly this is okay as well?

**Comment by MrPetovan** [comment]
I still don't think "comment" is relevant since it isn't just comments, activities like likes and attendance responses are also concerned.

And if "natural order" doesn't fit, I stand by "timeline order". I don't think we can say "newest posts" or "most recent posts" because reshared posts will be older than the posts around them. Although Twitter is using "Latests Tweets" to differentiate from their non-chronological order just called "Home".

**Comment by AndyHee** [comment]
This is how both US and GB translations are currently have this:


Screenshot_2019-11-02_19-15-18

**Comment by AndyHee** [comment]
`Posted order is now arrival order`

The term "order" is redundant, I'd say.

**Comment by AlfredSK** [comment]
Maybe "Latest Posts" and "Latest Activities"...
German "Neueste Beiträge" and "Neueste Aktivitäten"

**Comment by AndyHee** [comment]
> Maybe "Latest Posts" and "Latest Activities"...

Sounds good to me.

**Comment by tobiasd** [comment]
But *any* ordering of the timeline is a *timeline order*, isn't it.

File: mod/network.php [hunk 1]
```diff
@@ -925,19 +925,19 @@ function network_tabs(App $a)
 	// tabs
 	$tabs = [
 		[
-			'label'	=> L10n::t('Commented Order'),
-			'url'	=> str_replace('/new', '', $cmd) . '?order=comment' . (!empty($_GET['cid']) ? '&cid=' . $_GET['cid'] : ''),
+			'label'	=> L10n::t('Latest Activities'),
```
[the remark this answers is not in the document]
Comment (annando) [reply mod/network.php]: "Latest Activity" and "Latest Posts"?

[the remark this answers is not in the document]
Comment (MrPetovan) [reply mod/network.php]: Yes.

**Comment by MrPetovan** [comment]
> But _any_ ordering of the timeline is a _timeline order_, isn't it.

Yes, because the "timeline order" is the default one on Friendica and many other platforms, it is what is expected from a social media timeline.

File: mod/network.php [hunk 2]
```diff
@@ -925,18 +925,18 @@ function network_tabs(App $a)
...
+			'title'	=> L10n::t('Sort by lastest activity on the posts'),
 			'id'	=> 'activity-order-tab',
 			'accesskey' => 'e',
 		],
 		[
 			'label'	=> L10n::t('Latest Posts'),
 			'url'	=> str_replace('/new', '', $cmd) . '?order=post' . (!empty($_GET['cid']) ? '&cid=' . $_GET['cid'] : ''),
 			'sel'	=> $post_active,
-			'title'	=> L10n::t('Sort by post date'),
+			'title'	=> L10n::t("Sort by the posts' receiving date"),
```
Comment (MrPetovan) [inline mod/network.php]: `Sort by post received date` instead.

File: mod/network.php [hunk 3]
```diff
@@ -925,18 +925,18 @@ function network_tabs(App $a)
 	// tabs
 	$tabs = [
 		[
-			'label'	=> L10n::t('Latest Activities'),
+			'label'	=> L10n::t('Latest Activity'),
 			'url'	=> str_replace('/new', '', $cmd) . '?order=activity' . (!empty($_GET['cid']) ? '&cid=' . $_GET['cid'] : ''),
 			'sel'	=> $all_active,
-			'title'	=> L10n::t('Sort by activity Date'),
+			'title'	=> L10n::t('Sort by lastest activity on the posts'),
```
Comment (MrPetovan) [inline mod/network.php]: `Sort by latest activity (post, comment, like...)`

**Pull request merged**